### DISCUSSION TITLE
New version, new repo...

### DISCUSSION THREAD

**Discussion by albakhtari** [LinSoftWin/Photoshop-CC2022-Linux#26] [General]
Hello @MiMillieuh, I just wanted to share some advice with you (something that I'm going to do too): When you start working on a new version of Photoshop, I recommend that you create a separate repo for each version, this will make it easier for people searching for specific versions of Ps.
Hence, it is easier for you to develop the new version while your older ones are still accessible, and at the same time others can contribute with their recommendations or PRs separately.

**Comment by MiMillieuh** [comment]
> Hello @MiMillieuh, I just wanted to share some advice with you (something that I'm going to do too): When you start working on a new version of Photoshop, I recommend that you create a separate repo for each version, this will make it easier for people searching for specific versions of Ps.
> Hence, it is easier for you to develop the new version while your older ones are still accessible, and at the same time others can contribute with their recommendations or PRs separately.   

Hello, I keep everything in the same repo on purpose. My initial objective was to keep Photoshop as upstream as it is on windows.
When the user want to update, they know that they just have to go were they download the first version.

Also I'm not deleting old version and I keep support on it.

Especially for now as PS 2022 isn't ready for production usage.

It will be soon more clear in the repo
